<!--StartFragment-->

Summary written by: [Titilola Shittu(Lola)](https://github.com/lhorla), [Osama Shukri(Osama)](https://github.com/osamashukri), [Oluwatobiloba Osedimilehin(Tobijayyy)](https://github.com/tobijayyy), [Chukwuemeka Nwachuya(Emzy247)](https://github.com/Megzi247) 

[Watch video summary](https://www.linkedin.com/posts/titi-shittu_biomarkerdiscovery-hackbiointernship-lungcancerresearch-activity-7238173218757308416-QDHo?utm_source=share&utm_medium=member_desktop)

# **Development of a RNA-Seq Based Prognostic Signature in Lung Adenocarcinoma**

As of 2015, lung cancer was the leading cause of death in the United States. Due to the low survival rate associated with it, the need to predict the chances of patient survival is great. RNA-Seq-based prognostic signatures in lung adenocarcinoma represent a powerful approach for predicting disease outcomes by analyzing gene expression profiles. In this study, the researchers used RNA-seq data from The Cancer Genome Atlas (TCGA) and a patient cohort from the University of Michigan. The goal was to develop a gene signature that could predict patient survival outcomes to guide treatment decisions.


## **Validation of gene signature**

Researchers divided 412 lung adenocarcinoma cases from TCGA into a training cohort (255 cases) and a validation cohort (157 cases). They analyzed gene expression data using univariate and multivariable Cox analysis. 96 genes with significant prognostic associations were identified, including several long non-coding RNAs (lncRNAs). The genes were streamlined to 13, using Benjamini-Hochberg false discovery rate (FDR) and Hazard ratios (HRs),  from where a model with a group of 4 genes with the highest predictive ability emerged. This predictive gene set comprised **RHOV**, **CD109**, **FRRS1**, and **LINC00941**, which are overexpressed in Lung Adenocarcinoma.


## **Clinical implications**

The four-gene signature effectively stratified patients into high- and low-risk groups, demonstrating significant associations with overall survival (OS) and metastasis-free survival (MFS) in the primary TCGA and validation cohorts.  In TCGA cohort II, the signature achieved hazard ratios (HR) of 3.07 for OS and 3.05 for MFS, independent of age, sex, and stage. The Michigan University cohort showed similar results, with an HR of 2.05 for OS. The model predicted prognosis in early-stage lung adenocarcinoma patients (stage I) but was also effective for more advanced stages (II-IV). This shows both the TCGA dataset and the University of Michigan cohort consistently predicted poor outcomes in high-risk patients, regardless of age, sex, or cancer stage.

Further analysis revealed that high-risk tumors exhibited increased activity in cancer-related processes such as immune system function and blood vessel formation. The four-gene signature also strongly links to Epidermal Growth Factor Receptor (EGFR-related) processes, which are important in lung cancer biology.


## **Conclusion**

The study showed that RNA-seq expression data is useful for obtaining gene sets that can predict the survival of patients with lung adenocarcinoma. Limitations included the retrospective nature of the validation and small sample sizes for certain mutations, which restricted a full assessment of the signatureâ€™s performance in specific mutational cohorts. Ongoing research is needed to validate and integrate these signatures into routine clinical care.


## **Reference**

Sudhanshu Shukla, Joseph R Evans, Rohit Malik, Felix Y Feng, Saravana M Dhanasekaran, Xuhong Cao, Guoan Chen, David G Beer, Hui Jiang, Arul M Chinnaiyan, Development of a RNA-Seq Based Prognostic Signature in Lung Adenocarcinoma, _JNCI: Journal of the National Cancer Institute_, Volume 109, Issue 1, January 2017, djw200, <https://doi.org/10.1093/jnci/djw200>



<!--EndFragment-->
